dask/dataframe/io/csv.py

```diff
@@ -915,6 +915,8 @@ def to_csv(
         values = [value]
         files = [first_file]
     else:
+        if os.path.isdir(filename) and "*" not in filename:
+            filename = os.path.join(filename, "part-*.csv")
```

Comment:
`to_parquet` outputs files like `part.{index}.parquet`, perhaps we should do the same here for consistency?

```suggestion
            filename = os.path.join(filename, "part.*.csv")
```